modules/custom/az_marketing_cloud/config/install/views.view.az_marketing_cloud.yml

```diff
@@ -366,7 +366,7 @@ display:
           element_label_colon: true
           element_wrapper_type: ''
           element_wrapper_class: ''
-          element_default_classes: false
+          element_default_classes: true
```

Comment:
do you want this to be true?

Reply:
I don't think so, but what is best practice?